src/javascript/binary/common_functions/account_opening.js

```diff
@@ -182,12 +182,36 @@ const AccountOpening = (() => {
         return validations;
     };
 
+    const handleAccountType = (e = {}) => {
+        if(e.target && e.target.value === 'ico') {
+            professionalClient.init(false, false, true);
+            $('h1').html(localize('ICO Account Opening'));
+        } else {
+            document.getElementById('fs_professional').setVisibility(0);
+            $('h1').html(localize('Real Money Account Opening'));
+        }
+    };
+
+    const accountTypeInit = (form_id) => {
+        $(form_id)
+            // Show account type options
+            .find('#fieldset_account_type')[0].setVisibility(1);
+        $(form_id)
+            .find('#fieldset_account_type input[type=radio]')
```

Comment:
can chain this with above, no need to find the same things again. `$(form_id).find('#fieldset_account_type').setVisibility(1).find('input[type=radio]')`